src/Output/Output_L1Error.cpp

```diff
@@ -187,7 +204,19 @@ void Output_L1Error( void (*AnalFunc_Flu)( real fluid[], const double x, const d
                   EdgeL  = amr->patch[0][lv][PID]->EdgeL;
                   EdgeR  = amr->patch[0][lv][PID]->EdgeR;
 
-                  if ( Part == OUTPUT_DIAG ) // (+1,+1,+1) diagonal
+                  if ( Part == OUTPUT_BOX )
+                  {
+//                   check whether the cell is within the target range
+                     for (int k=0; k<PS1; k++)  {
+                     for (int j=0; j<PS1; j++)  {
+                     for (int i=0; i<PS1; i++)  {
+
+                        WriteFile( AnalFunc_Flu, AnalFunc_Mag, File, lv, PID, i, j, k, L1_Err, Part );
+
+                     }}} // i, j, k
+                  } // if ( Part == OUTPUT_BOX )
```

Comment:
This code snippet seems unnecessary, as you've already set `Check_x = Check_y = Check_z = false` for `OUTPUT_BOX`, allowing it to be treated the same as, for example, `OUTPUT_X`. How about removing it to reduce redundant code?